src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -365,15 +365,15 @@ internal static void ValidateIssuerSecurityKey(SecurityKey securityKey, Security
             if (validationParameters.IssuerSigningKeyValidatorUsingConfiguration != null)
             {
                 if (!validationParameters.IssuerSigningKeyValidatorUsingConfiguration(securityKey, securityToken, validationParameters, configuration))
-                    throw LogHelper.LogExceptionMessage(new SecurityTokenInvalidSigningKeyException(LogHelper.FormatInvariant(LogMessages.IDX10232, securityKey)) { SigningKey = securityKey });
+                    throw LogHelper.LogExceptionMessage(new SecurityTokenInvalidSigningKeyException(LogHelper.FormatInvariant(LogMessages.IDX10232, LogHelper.MarkAsNonPII(securityKey.ToString()))) { SigningKey = securityKey });
```

Comment:
@GeoK double checking that logging the Kid and InternaId is acceptable? [SecurtiyKey.ToString()](https://learn.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.tokens.securitykey.tostring?view=msal-web-dotnet-latest#microsoft-identitymodel-tokens-securitykey-tostring) returns the formatted string: GetType(), KeyId: 'value', InternalId: 'value'.

Reply:
Otherwise, I can log Key.Kid + change all the log messages from `.. key: {0}..` to `.. key Kid: {0}..`

Reply:
Was the same returned previously? The difference is that now we are not treating it as PII.

Reply:
Can confirm that while debugging, logging `SecurityKey` and `SecurityKey.ToString()` both invoke .ToString() and have the same output.

Reply:
And it's acceptable to also log `InternalId`?

Reply:
Discussed in internally offline